src/client.ts

```diff
@@ -2317,6 +2318,23 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         return this.crypto.getStoredDevicesForUser(userId) || [];
     }
 
+    /**
+     * Get the device information for the given list of users.
+     *
+     * @param userIds - The users to fetch.
+     * @param downloadUncached - If true, download the device list for users whose device list we are not
+     *    currently tracking. Defaults to false, in which case such users will not appear at all in the result map.
+     *
+     * @returns A map `{@link DeviceMap}`.
+     */
+    public getUserDeviceInfo(userIds: string[], downloadUncached = false): Promise<DeviceMap> {
+        if (!this.cryptoBackend) {
+            throw new Error("End-to-end encryption disabled");
+        }
+
+        return this.cryptoBackend.getUserDeviceInfo(userIds, downloadUncached);
+    }
+
```

Comment:
please don't add more methods to `MatrixClient`. callers should call `getCryptoApi.getUserDeviceInfo`